service/src/LevelZeroIOGroup.cpp

```diff
@@ -1010,7 +1044,24 @@ namespace geopm
             catch (const geopm::Exception &ex) {
                 throw Exception("LevelZeroIOGroup::" + std::string(__func__) + ": "
                                 + " Failed to fetch frequency control range for "
-                                " GPU_CHIP domain " + std::to_string(domain_idx),
+                                " GPU_CHIP domain " + std::to_string(domain_idx) +
+                                "due to exception.  Exception: " + ex.what(),
```

Comment:
Suggestion: "due to exception: " + ex.what()

Same suggestion applies in changes below.

Disregard if this is already an established pattern followed for many exceptions in the file or LevelZero code in general.